src/time.cr

```diff
@@ -84,7 +84,7 @@ require "crystal/system/time"
 #
 # ```
 # time = Time.local(2018, 3, 8, 22, 5, 13, location: Time::Location.load("Europe/Berlin"))
-# time          # => 2018-03-08 22:05:13 +01:00 Europe/Berlin
+# time.to_s     # => "2018-03-08 22:05:13 +01:00"
```

Comment:
This change looks odd. In the next example there's also just `time # =>` with its inspect value.

Reply:
Nice catch! You're right. 👍 

This was a transitional fix before I supported new time literals and forgot to revert.
Updated(Reverted).